### PR TITLE
Decompose and organize `import-tests` script

### PR DESCRIPTION
**Pull request by stalgiag**
This pull request reorganizes and documents the code in the import-tests script. I tried to avoid changing functionality and only made minor code changes beyond breaking big functions up, grouping related functions into specific modules, and add JSDoc comments.

The database builds and the tests pass. I assume that this script is so foundational to DB construction that at least one of the tests would surface an errant change in functionality. If this is not the case, and there is room for this to introduce breaking changes, then I am okay with not merging out of an abundance of caution.

I also recognize that this reorganization is highly damaging to the git history. If we don't feel that the tradeoff in readability/maintainability is worth it then I would be happy to discard this PR.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207768136984532

### CODE REVIEW

File: server/scripts/import-tests/gitOperations.js
Comment (howard-e) [inline server/scripts/import-tests/gitOperations.js:79]: Doesn't look like this is getting used outside this function so no need to export (and perhaps for the best to keep the operations contained and true to the file's purpose)

File: server/scripts/import-tests/settings.js
Comment (howard-e) [inline server/scripts/import-tests/settings.js:3]: This also reminds me, could you include `server/scripts/import-tests/tmp` in the `.gitignore`?

I've been guilty in the past of accidentally including it in a `git add .` (but caught it!) while an import was still happening. It's such a quick process that I suppose it's been missed all this time

File: server/scripts/import-tests/testParser.js
Comment (howard-e) [inline server/scripts/import-tests/testParser.js:15]: Neat!

File: server/scripts/import-tests/testParser.js
Comment (howard-e) [inline server/scripts/import-tests/testParser.js:20]: nit: continuining to use the v1 identifier in the function names could be pretty useful to not show any "default" preference or priority.

File: server/scripts/import-tests/testPlanVersionOperations.js
Comment (howard-e) [inline server/scripts/import-tests/testPlanVersionOperations.js:480]: These 2 also aren't being called elsewhere

**Review by howard-e — COMMENTED** [review]
@stalgiag this is such a good separation of concerns! Thanks much! I've left a few inline comments and suggestions but nothing blocking

**Review by howard-e — APPROVED** [review]
@stalgiag this looks good to me!

> DB construction that at least one of the tests would surface an errant change in functionality. If this is not the case, and there is room for this to introduce breaking changes

Yep a test should exist to confirm the state of the TestPlanVersion rows is the same before and after an import.

The other tests not breaking still serve as an indicator that all _should_ be well given they're built around the import script but an explicit test like above should be done. It can be done in this PR or in another, I don't think this has to be blocked on that.

**Comment by stalgiag** [comment]
Adding a note to wait to merge this until after additional testing from #1169 and #1175 is merged in and confirmed

File: server/scripts/import-tests/settings.js
Comment (stalgiag) [reply server/scripts/import-tests/settings.js:3]: Done!

**Pull request merged**
